packages/kit/src/views/Market/MarketDetailV2/components/SwapPanel/components/TokenInputSection/TokenList.tsx

```
@@ -2,66 +1,201 @@
+          dt.contractAddress === iToken.contractAddress,
+      );
+
+      const priceToUse = dToken?.price;
+      const balanceToUse = dToken?.balance;
+
+      const networkConfig = Object.values(presetNetworksMap).find(
+        (n) => n.id === iToken.networkId,
+      );
+
+      const valueProps =
+        priceToUse && parseFloat(priceToUse) > 0
```

Biome (1.9.4)

[error] Use Number.parseFloat instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseFloat instead.


(lint/style/useNumberNamespace)